<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InputPatchEditor</class>
 <widget class="QDialog" name="InputPatchEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>434</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Input patch editor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0" colspan="2">
    <widget class="QTabWidget" name="m_tab">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="Mapping">
      <attribute name="title">
       <string>Mapping</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0" colspan="2">
        <widget class="QTreeWidget" name="m_mapTree">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <column>
          <property name="text">
           <string>Inputs</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="1" column="0" rowspan="3">
        <widget class="QTextBrowser" name="m_infoBrowser">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QToolButton" name="m_configureButton">
         <property name="text">
          <string notr="true">...</string>
         </property>
         <property name="icon">
          <iconset resource="main.qrc">
           <normaloff>:/configure.png</normaloff>:/configure.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>26</width>
           <height>26</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>109</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="0">
        <widget class="QCheckBox" name="m_feedbackEnabledCheck">
         <property name="toolTip">
          <string>Send feedback data to the input universe for moving e.g. motorized sliders.</string>
         </property>
         <property name="text">
          <string>Feedback enabled</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QToolButton" name="m_reconnectButton">
         <property name="toolTip">
          <string>Attempt to re-connect the current plugin</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="main.qrc">
           <normaloff>:/detach.png</normaloff>:/detach.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>26</width>
           <height>26</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Profile">
      <attribute name="title">
       <string>Profile</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0" rowspan="4">
        <widget class="QTreeWidget" name="m_profileTree">
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <property name="itemsExpandable">
          <bool>false</bool>
         </property>
         <property name="allColumnsShowFocus">
          <bool>true</bool>
         </property>
         <column>
          <property name="text">
           <string>Profile</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QToolButton" name="m_addProfileButton">
         <property name="toolTip">
          <string>Create a new input profile</string>
         </property>
         <property name="text">
          <string notr="true">...</string>
         </property>
         <property name="icon">
          <iconset resource="main.qrc">
           <normaloff>:/edit_add.png</normaloff>:/edit_add.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>26</width>
           <height>26</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QToolButton" name="m_removeProfileButton">
         <property name="toolTip">
          <string>Delete the selected input profile</string>
         </property>
         <property name="text">
          <string notr="true">...</string>
         </property>
         <property name="icon">
          <iconset resource="main.qrc">
           <normaloff>:/edit_remove.png</normaloff>:/edit_remove.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>26</width>
           <height>26</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QToolButton" name="m_editProfileButton">
         <property name="toolTip">
          <string>Edit the selected input profile</string>
         </property>
         <property name="text">
          <string notr="true">...</string>
         </property>
         <property name="icon">
          <iconset resource="main.qrc">
           <normaloff>:/edit.png</normaloff>:/edit.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>26</width>
           <height>26</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>170</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDialogButtonBox" name="m_buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QRadioButton" name="m_editorUniverseRadio">
     <property name="toolTip">
      <string>Use this universe for e.g. scene channel value setting in scene editor</string>
     </property>
     <property name="text">
      <string>Default editor universe</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="main.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>m_buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>InputPatchEditor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>424</x>
     <y>454</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>InputPatchEditor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>424</x>
     <y>454</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_mapTree</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>InputPatchEditor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>430</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_feedbackEnabledCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>InputPatchEditor</receiver>
   <slot>slotFeedbackToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>389</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>424</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotFeedbackToggled(bool)</slot>
 </slots>
</ui>
